package.json

```diff
@@ -11,20 +11,21 @@
   },
   "private": true,
   "dependencies": {
+    "@100mslive/hms-video": "^0.9.28",
```

Comment:
this package doesn't need to be added, 100ms deprecated this last year I believe